src/Apps/Collect/Components/Collection/CollectionFilterContainer.tsx

```diff
@@ -67,7 +67,10 @@ export const CollectionFilterFragmentContainer = createFragmentContainer(
             defaultValue: [MEDIUM, MAJOR_PERIOD, TOTAL]
           }
         ) {
-        artworks(aggregations: $aggregations) {
+        artworks(
+          aggregations: $aggregations
+          include_medium_filter_in_aggregation: true
```

Comment:
👍 this will only make sense when in the context of a 'gene' filter (mediums are genes). Since collections are also mixed bags of filter params (which will always include a gene/medium right?), this seems appropriate for the Collection app/component.